esmvaltool/utils/cmorizers/obs/cmorize_obs_CDS-SATELLITE-SOIL-MOISTURE.ncl

```diff
@@ -0,0 +1,137 @@
+; #############################################################################
+; ESMValTool CMORizer for CDS-SATELLITE-SOIL-MOISTURE data
+; #############################################################################
+;
+; Tier
+;    Tier 2: other freely-available dataset.
+;
+; Source
+;    https://cds.climate.copernicus.eu/cdsapp#!/dataset/satellite-soil-moisture?tab=form
+;
+; Last access
+;    20190314
+;
+; Download and processing instructions
+;    - Download the data (daily files) from source
```

Comment:
Do you mean "Day average"?
Please also add that all years, months and days have to be selected, which Type of Sensor, Type of Record and Version have to be selected from the source.

Reply:
Thanks for the review. I pushed a new commit.

Reply:
I am now making a test plot, and find out that I have to change a few things, working on it, and will let you know when I think the script is finished.

Reply:
I changed the MIP to 'day', since this is daily data. Also I gave VERSION a more meaningful name. It now reads 'COMBINED-TCDR-v201812.0.0' (is it ok to have dots in this string?). @hb326  I think it would be good if you give more guidance on how we name our datasets, just to get the same approach for each ECV. Many datasets on the CDS have both a CDR and TCDR version, a uniform approach would be nice, right?

Reply:
What do you think? It would be nice to have this in version2_development, such that I can delete one of my many branches ;-)